.github/workflows/main.yml

```diff
@@ -1,20 +1,43 @@
 # https://docs.cypress.io/guides/continuous-integration/github-actions
 
-name: Cypress integration tests
+name: main CI jobs (integration tests)
 
 on: push
 
 jobs:
 
+  install:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Cypress install (and build app)
+        uses: cypress-io/github-action@v6
+        with:
+          build: npm run build
```

Comment:
This works but is way overkill for what our tests are doing, because currently we're only using a handful of pages in the test suite.